api/models/process.go

```diff
@@ -217,6 +218,7 @@ func ListProcesses(app string) ([]*Process, error) {
 		}
 
 		if ec2i == nil {
+			// Error out if it didn't find an ec2 instance? Maybe just continue?
```

Comment:
I think the behavior of a total error of ECS or EC2 API calls fail is ok for now. All the data points to the Docker parts being the main problem.

Though I think that the best design would be saying the ECS APIs are canonical and even EC2 data is nice to have.